tests/argv.php

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/php -q
```

Comment:
Nope. Good point, I totally spaced out here. 

Before trying to come up with something dumb/clever I'm going to wait to see if anyone has a better idea on how to solve this problem.